UPGRADE.md

```diff
@@ -0,0 +1,8 @@
+UPGRADE FROM 4.0 to 5.0
+=======================
+
+ScriptHandler
+-------------
```

Comment:
Well, we don't need sub-sections inside the upgrade dir IMO (we have them in Symfony to split by component because the framework is huge, but this bundle is very small)